health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -0,0 +1,647 @@
+    let activeCounters = { top: 0 }; // Track active question counts at each level
+
+    // Helper function to generate codes recursively
+    const generateCode = (question, prefix, index, parentCounter = '') => {
+      // Generate code regardless of isActive status
+      let code = '';
+      if (question.parentId === null) {
+        code = `SN${index + 1}`;
+        // Only increment counter for active questions
+        if (question.isActive) {
+          activeCounters.top += 1;
+          parentCounter = String(activeCounters.top);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)